src/akamai/js/akamaiHD_flash.js

```diff
@@ -564,7 +564,7 @@ require("../../../html5-common/js/utils/constants.js");
 
     var raiseBitratesAvailable = function(event) {
       var vtcBitrates = [{id: "auto", width: 0, height: 0, bitrate: 0 }];
-      for (var i in event.eventObject) {
+      for (var i = 0; i < event.eventObject.length; i++) {
         var vtcBitrate = {
           id: event.eventObject[i].id,
```

Comment:
Has been changed some days back, githib is failing to update the conversation page to reflect the current code...